### PR TITLE
feat(callout): add accordion and popover callouts

### PR DESCRIPTION
**Pull request by Supamiu**
## Description

Add accordion and popover callouts

-----

-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2413/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/9590/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2413/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/9597/ui-diff)

**Review by jbosi — APPROVED** [review]

**Review by jeremie-lucca — APPROVED** [review]

File: packages/ng/callout/callout-feedback-list/callout-feedback-list.component.scss [hunk 1]
```diff
@@ -0,0 +1 @@
+@use '@lucca-front/scss/src/components/calloutFeedbackList';
```
Comment (GuillaumeNury) [inline packages/ng/callout/callout-feedback-list/callout-feedback-list.component.scss:1]: We use the same stylesheet twice? The style will be duplicated in the DOM. Could we split `@use '@lucca-front/scss/src/components/calloutFeedbackList'` In two files?

**Comment by shajz** [comment]
That's a lot of commits, is there something wrong with the base branch ? Could you rebase ?

**Comment by Supamiu** [comment]
> That's a lot of commits including merged PRs, is there something wrong with the base branch ? Could you rebase to have a cleaner, less confusing history ?

This is due to the mess that happened with `rc`, I'd like to avoid having to filter commits because most of them are for SCSS which I didn't make myself, the history being erased into a single squashed commit in the end, I don't think it matters that much.

File: packages/ng/callout/callout-feedback-list/callout-feedback-list.component.scss (hunk 1 above)
Comment (Supamiu) [reply packages/ng/callout/callout-feedback-list/callout-feedback-list.component.scss:1]: ping @jeremie-lucca (it's used in both list and item components)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2413/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/9658/ui-diff)

**Review by jeremie-lucca — APPROVED** [review]

**Review by GuillaumeNury — APPROVED** [review]

**Pull request merged**